test/testenv/secretutil.go

```diff
@@ -158,8 +168,14 @@ func UpdateSecret(deployment *Deployment, ns string, secretObj SecretResponse) (
 		logf.Log.Error(err, "Failed to parse response")
 		return false, err
 	}
-	modify := ModifySecretObject(deployment, data, ns, secretName)
-	return modify, err
+	// Update or delete the secret object based on delete parameter
+	update := false
+	if delete {
```

Comment:
Add a log here to specify the operation. 
`logf.Log.Info("Deleting the secret object", "Secret Name", SecretName)`

Reply:
Logs present in called function